examples/basic/variables.tf

```diff
@@ -39,3 +39,15 @@ variable "resource_tags" {
   description = "Optional list of tags to be added to created resources"
   default     = []
 }
+
```

Comment:
Ideally we don't want to expose new variables in examples like this, and instead hard code values in the examples. However I see why you did it in order to use same example for different tests. But if we are able to use pre-existing tests / examples instead as per my comments, then maybe you can remove these variables from here